llm/llama-2/README.md

```diff
@@ -0,0 +1,94 @@
+# Self-Hosted LLaMA 2 Chatbot on Any Cloud
+
+[LLaMA 2](https://github.com/facebookresearch/llama/tree/main) is the top open-source models on the [Open LLM leaderboard](https://huggingface.co/spaces/HuggingFaceH4/open_llm_leaderboard) today. It has been released with a license that authorizes commercial use. You can deploy a private Llama 2 chatbot with SkyPilot in your own cloud with just one simple command.
+
+* [LLaMA 2 release](https://github.com/facebookresearch/llama/tree/main)
+* [LLaMA 2 paper](https://ai.meta.com/research/publications/llama-2-open-foundation-and-fine-tuned-chat-models/)
+
+## Why use SkyPilot to deploy over commercial hosted solutions?
+
+* No lock-in: run on any supported cloud, region
+* Everything stays in your cloud account (your own VMs, cloud object store buckets)
+* No one else sees your chat history
+* Pay absolute minimum (no managed solution markups)
+* Freely choose your own model size, GPU type, number of GPUs, etc, based on needs and budget.
+
+…and you get all of this with 1 click — let SkyPilot automate the infra.
+
+## Pre-requisites
+
+###  Step 1: Apply for the access to the LLaMA 2 model
+
+Go to the [application page](https://ai.meta.com/resources/models-and-libraries/llama-downloads/) and apply for the access to the model weights.
+
+
+### Step 2: Get the access token from huggingface
```

Comment:
Nit: Should we have these as bullet points instead of headlines? (Feel free to pick whichever looks better to you)